views/pages/registeruser.ejs

```diff
@@ -71,6 +71,35 @@
               value="<%= typeof city != 'undefined' ? city : '' %>"
               style="border: solid grey;"
             /><br />
+
+            <p id="preferenceser" style="color: red;"></p>
+            <label for="preferences">Choose a Preferences:</label>
+            <select id="preferences" name="preferences">
+            <option value="breakfast">Breakfast</option>
+            <option value="lunch">Lunch</option>
+            <option value="dinner">Dinner</option>
+            </select> <br/>
+
+            <p id="preferenceser2" style="color: red;"></p>
+            <label for="preferences2">Choose a Preferences:</label>
```

Comment:
are these all the cuisines?

Reply:
there are many possible preferences a person, could have selected if i provided all of them, I thought if i narrowed it down, it would be more simple and easy to select for a user